graylog2-server/src/main/java/org/graylog2/lookup/LookupTable.java

```diff
@@ -71,6 +71,9 @@ public String error() {
     public LookupResult lookup(@Nonnull Object key) {
         final LookupResult result = cache().get(LookupCacheKey.create(dataAdapter(), key), () -> dataAdapter().get(key));
 
+        if (result.hasError()) {
+            return result;
```

Comment:
Note to self: This introduces a change of behavior. Lookups with errors will not return a default value anymore